ui/src/components/WMWorkerPoolsTable/index.jsx

```diff
@@ -305,6 +305,7 @@ export default class WorkerManagerWorkerPoolsTable extends Component {
             String(workerPool.providerId).includes(filterValue) ||
             String(workerPool.owner).includes(filterValue)
           }
+          {...this.props}
```

Comment:
It looks like this component just needs the `history` prop -- perhaps better to only pass that single property along?

Reply:
ok, I'll change it.
Thanks